checker/bin-devel/javac

```diff
@@ -35,7 +35,7 @@ annoToolsDir="${cfDir}"/../annotation-tools
 # Put afu jar files last, as they might contain out-of-date CF files.
 # Put "checker" after the other sub-projects, as "ant bindist" puts
 # other projects into the checker/build directory.
-buildDirs="${cfDir}"/dataflow/build:"${cfDir}"/javacutil/build:"${cfDir}"/stubparser/build:"${cfDir}"/framework/build:"${cfDir}"/checker/build:"${annoToolsDir}"/scene-lib/bin:"${annoToolsDir}"/annotation-file-utilities/annotation-file-utilities.jar
+buildDirs="${cfDir}"/dataflow/build:"${cfDir}"/javacutil/build:"${cfDir}"/framework/build:"${cfDir}"/checker/build:"${cfDir}"/checker/lib/javaparser-core.jar:"${annoToolsDir}"/scene-lib/bin:"${annoToolsDir}"/annotation-file-utilities/annotation-file-utilities.jar
```

Comment:
@panacekcz , the pull request removed the stub parser source code from the Checker Framework repository and added, in its place, a dependency on a .jar file built from an external repository.  So, adding that dependence is correct.

@feliperodri , should we rename the `javaparser-core.jar` file to `stubparser-core.jar`, to emphasize that this is not the real JavaParser but our modified version of it?  (However, all the package names are not changed, so it isn't currently possible to use real JavaParser at the same time as the Stub Parser.)